src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -138,7 +138,7 @@ $modal-layer: 400;
 }
 
 .merger #bg-middle-effect {
-  background: url(/va_files/2014/responsive/images/bg-middle-effect.png) repeat-y scroll center -65px $teamsite-light-blue;
+  background: url(/img/teamsite/bg-middle-effect.png) repeat-y scroll center -65px $teamsite-light-blue;
```

Comment:
Was the `va_files` link being processed into an S3 bucket URL and hitting a 404? We added an [exception](https://github.com/department-of-veterans-affairs/vets-website/blob/master/script/pre-archive/link-assets-to-bucket.js#L65) for files with `va_files` in their path to load through the domain instead of the bucket, so the original link should work now. Not sure if there's a slight difference between images or something though.

Reply:
No, it sounds like this PR is no longer necessary, thanks @ncksllvn. Closing.

Reply:
va_files is not part of our hosting- it's on the team site. I think this bg was reused on one teammate but it's not present on all of the pages.

Reply:
Thanks @rianfowler, reopened.

Reply:
Is there any page we can visit to test this? @rianfowler we allow files hosted at `https://www.va.gov/va_files` to use the domain.

Reply:
www.benefits.va.gov tries to pull it from two locations but I don't know why or where other than that css file. I think there might be something in the team site page itself using it

Reply:
Linking to our Slack [convo](https://dsva.slack.com/archives/C7S6EA0ES/p1541647446521500). Looks like we do need this file added if we want to load this background image, or do some work with the `pre-archive` step of our build.

Or another option is to propose using VA.gov's homepage background on TeamSite pages.